cmd/controller/main.go

```diff
@@ -165,6 +166,12 @@ func main() {
 			coreServiceInformer,
 			virtualServiceInformer,
 		),
+		labeler.NewController(
```

Comment:
It is not very clear from the package name that this only labels Configuration with Route.

May be config_labeler?

Reply:
I changed the constructor name and added a package comment in doc.go